app/controllers/api/users.php

```diff
@@ -355,9 +355,6 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             $queries[] = Query::search('search', $search);
         }
 
-        // Set default limit
-        $queries[] = Query::limit(25);
-
```

Comment:
why dont we wanna apply our default limits here?

Reply:
No need to, from PR reviews on another branch. Limit 25 is there by default from library, no need to be explicit in every action.